tests/parser_templates/cli/show_interfaces/main.yaml

```diff
@@ -18,9 +18,9 @@
       - interfaces['GigabitEthernet0/0/1']['admin-status'] == 'enabled'
       - interfaces['Tunnel0']['admin-status'] == 'enabled'
       - interfaces['Tunnel0']['oper-status'] == 'down'
-      - interfaces['GigabitEthernet0']['counters']['in-broadcast-pkts'] == '4891144'
+      - interfaces['GigabitEthernet0']['counters']['in-broadcast-pkts'] | int == 4891144
       - interfaces['GigabitEthernet0']['counters']['in-multicast-pkts'] == None
-      - interfaces['Virtual-Access1']['mtu'] == '1500'
+      - interfaces['Virtual-Access1']['mtu'] | int == 1500
```

Comment:
same comment

Reply:
I think the output of these parsers should be integers for straight numbers. 

This may be a problem with `command_parser`, as I haven't been able to get it to return int types, even when using jinja2 filters.

I'll revert these for now to get the other stuff in, but these should be moved to integers at some point.